cupy/creation/ranges.py

```diff
@@ -126,7 +126,66 @@ def logspace(start, stop, num=50, endpoint=True, base=10.0, dtype=None):
     return core.power(base, y).astype(dtype)
 
 
-# TODO(okuta): Implement meshgrid
+def meshgrid(*xi, **kwargs):
+    """Return coordinate matrices from coordinate vectors.
+
+    Given one-dimensional coordinate arrays x1, x2, ..., xn, this function
+    makes N-D grids.
+
+    For ndarrays x1, x2, ..., xn with lengths ``Ni = len(xi)``, return
```

Comment:
For consistency with the previous sentence, describing "For one-dimensional arrays ..." looks better to me. Further, writing "this function returns ..." seems better than just "return".